lib/update/cluster/phases/init.go

```diff
@@ -270,6 +282,30 @@ func (p *updatePhaseInit) updateClusterDNSConfig() error {
 	return nil
 }
 
+// updateClusterInfoMap updates the cluster info config map or creates it
+// if it doesn't exist yet.
+func (p *updatePhaseInit) updateClusterInfoMap() error {
+	p.Infof("Update %v config map.", constants.ClusterInfoMap)
+	_, err := p.Client.CoreV1().ConfigMaps(constants.KubeSystemNamespace).Get(
+		constants.ClusterInfoMap, metav1.GetOptions{})
+	if err == nil {
+		p.Info("Config map %v already exists.", constants.ClusterInfoMap)
+		return nil
+	}
+	err = rigging.ConvertError(err)
+	if !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+	// Cluster info config map doesn't exist yet, create it.
+	configMap := phases.ClusterInfoMap(ops.ConvertOpsSite(p.Cluster))
```

Comment:
Not to be too nit-picky, but it's puzzling that lib/update is using lib/install for this - I'd think that this configuration should be stored in another package which both lib/install and lib/update could use. Could be even a totally new package like `lib/kubernetes` or `lib/config` or lib/cluster for Gravity-specific cluster configuration.